content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -368,15 +367,12 @@ to `Retain`, including cases where you are reusing an existing PV.
 Support for expanding PersistentVolumeClaims (PVCs) is enabled by default. You can expand
 the following types of volumes:
 
-* azureDisk
-* azureFile
-* awsElasticBlockStore
-* cinder (deprecated)
+* azureFile (deprecated)
 * {{< glossary_tooltip text="csi" term_id="csi" >}}
 * flexVolume (deprecated)
-* gcePersistentDisk
+* gcePersistentDisk (deprecated)
 * rbd
```

Comment:
rbd is deprecated

Reply:
rbd is deprecated in v1.28 https://github.com/kubernetes/kubernetes/pull/118552